apps/firehose_to_es_processor/test/test_util.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from lib.util import *
+
+
+class TestUtil(unittest.TestCase):
+
+    def test_extract_json_bounds(self):
+        # Should extract bounds of json if found
+        # test_input = "blue{'hi': 'hello'}blueeee"
+        # test_output = extract_json_bounds(test_input)
+        # self.assertEqual(test_output, (4, 18))
+
+        # Should not return bounds if no json is found
+        test_input = "blueeeeee"
+        test_output = extract_json_bounds(test_input)
+        self.assertEqual(test_output, (None, None))
+
+        # Should return bounds for json with internal double quote strings
+        test_input = 'blue{"hi": "hello"}blueeee'
+        test_output = extract_json_bounds(test_input)
+        self.assertEqual(test_output, (4, 18))
+
+        # Should extract bounds even if brackets wrap invalid json
+        test_input = 'blue{"hi"}blueeee'
+        test_output = extract_json_bounds(test_input)
+        self.assertEqual(test_output, (4, 9))
+
+        # Should extract outer bounds if there is nested json
+        test_input = "blue{'hi': {'hi': hello'}}blueeee"
```

Comment:
This test is invalid now, right?

Reply:
This test is still valid, as it is not the responsibility of extract_json_bounds to determine json validity.

Reply:
Ok, cool